tools/check_release.py

```diff
@@ -362,11 +394,22 @@ def get_latest_library_versions(repo_path):
 
     return mbed, mbed_dev
 
+def log_results(lst, title):
+    logging.info(title)   
```

Comment:
@theotherjimmy I don't understand your comment ? lst is checked for being empty (in which case output 'none', otherwise we need to iterate through the list printing entries . Are you suggesting printing title on every list entry line ?
E.g something more like:
FAILED: example_app
FAILED: other_example_app
SKIPPED: this_app 
?